src/main/java/com/eleks/academy/whoami/core/action/PlayerAction.java

```diff
@@ -0,0 +1,20 @@
+package com.eleks.academy.whoami.core.action;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
```

Comment:
`@Data` contains a lot of methods that will be generated do you use all of them? if not please put only those that you use